packages/web/src/components/edit-folder-modal/EditFolderModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback } from 'react'
+
+import {
+  IconFolder,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  ModalTitle
+} from '@audius/stems'
+import { useDispatch } from 'react-redux'
+
+import { PlaylistLibraryFolder } from 'common/models/PlaylistLibrary'
+import { getPlaylistLibrary } from 'common/store/account/selectors'
+import FolderForm from 'components/create-playlist/FolderForm'
+import {
+  getFolderId,
+  getIsOpen
+} from 'store/application/ui/editFolderModal/selectors'
+import { close } from 'store/application/ui/editFolderModal/slice'
+import { renamePlaylistFolderInLibrary } from 'store/playlist-library/helpers'
+import { update as updatePlaylistLibrary } from 'store/playlist-library/slice'
+import { useSelector } from 'utils/reducer'
+import { zIndex } from 'utils/zIndex'
+
+import styles from './EditFolderModal.module.css'
+
+const messages = {
+  editPlaylistFolderModalTitle: 'Edit Folder'
+}
+
+const EditFolderModal = () => {
+  const isOpen = useSelector(getIsOpen)
+  const folderId = useSelector(getFolderId)
+  const playlistLibrary = useSelector(getPlaylistLibrary)
+  const folder =
+    playlistLibrary == null || folderId == null
+      ? null
+      : (playlistLibrary.contents.find(
+          item => item.type === 'folder' && item.id === folderId
+        ) as PlaylistLibraryFolder | undefined)
```

Comment:
just wondering on the cast here since i think find should be able to infer the type here?

Reply:
ahh gotcha, thanks for the info, sounds good then!